pkg/reconciler/route/route.go

```diff
@@ -573,3 +575,24 @@ func setTargetsScheme(rs *v1alpha1.RouteStatus, dnsNames []string, scheme string
 		}
 	}
 }
+
+func buildIngressSplitsForChallenges(domains []string, challenges []netv1alpha1.HTTP01Challenge) []netv1alpha1.IngressRule {
+	splits := []netv1alpha1.IngressRule{}
+	for _, challenge := range challenges {
+		splits = append(splits, *resources.MakeIngressRuleWithPath(
+			domains,
+			challenge.ServiceNamespace,
+			false,
```

Comment:
Why do we pass in `false` for `isClusterLocal` here? Or should we pass the result of `domains.IsClusterLocal(domain)` in here?

Reply:
Also comment what does this _false_ signify  `false, /* isClusterLocal */`.

Reply:
I will add a comment there. It's `false` because all of the cluster local domains are removed from the tag map before the desired certificates are created. This function is called while iterating over those desired certificates.